src/main/java/uk/ac/cam/cl/dtg/isaac/dao/PgQuizQuestionAttemptPersistenceManager.java

```diff
@@ -75,4 +81,35 @@ public void registerQuestionAttempt(Long quizAttemptId, QuestionValidationRespon
             throw new SegueDatabaseException("Unable to process json exception", e);
         }
     }
+
+    @Override
+    public Map<String, List<QuestionValidationResponse>> getAnswers(Long quizAttemptId) throws SegueDatabaseException {
```

Comment:
Could this be clearer, `getAllAnswersForQuizAttempt` or something?

Reply:
Sure, can do.